drivers/misc/coresight/Kconfig

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_NORDIC_NRF_TBM := nordic,nrf-tbm
+
+config NRF_ETR
```

Comment:
is `coresight` a good name for this driver if it only has `NRF` stuff in here? Could this not just be `nrf_etr` or `nrf_tbm`? The drivers/misc/CMakeList.txt entry also looks super generic including some `coresight` folder where as you could probably just do `add_subdirectory_ifdef(CONFIG_NRF_ETR nrf_etr)`

Reply:
i discussed it with Carles and Gerard and we settled on that. This is a "driver" for coresight peripheral (ETR) but with nordic extensions so imo that is a good place. Changed Kconfig string to `Coresight ETR handler (with Nordic TBM)`.